src/Extensions/Extensions.jl

```diff
@@ -21,10 +21,16 @@ spliced back into the docstring in place of it:
 
     \\!!get(author)
 
-When no field in found in the metadata for the object, the module and package
+When no field is found in the metadata for the object, the module and package
 metadata are searched in turn.
 """
-Formats.metamacro(::META"get", body, mod, obj) =  Cache.findmeta(mod, obj, symbol(body))
+function Formats.metamacro(::META"get", body, mod, obj)
+    try
+        Cache.findmeta(mod, obj, symbol(body))
+    catch
+        "ERROR: No metadata found for ':$(symbol(strip(body)))'"
```

Comment:
I'd prefer to have these kind of things be show-stopping errors, rather than splicing in an error message. That goes for my `!!include` as well. There's no real reason a package author should let an undefined metadata field or missing file into a tagged version, so forcing them to resolve it sooner rather than later would be best in my opinion.

We can still produce better error messages though to help them. Include the file/line number of docstring as well as the contents of the metamacro that has an error.